proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,131 @@
+# Kubeflow Steering Committee
+The Kubeflow Steering Committee (KSC) is the governing body of the Kubeflow project, providing decision-making and oversight pertaining to the Kubeflow project policies, sub-organizations, and financial planning, and defines the project values and structure.
+
+The governance of Kubeflow is an open, living document, and will continue to evolve as the community and project change.
+
+### Charter
+
+1. Define, evolve, and promote the vision, values, and mission of the Kubeflow project.
+1. Define and evolve project and group governance structures and policies, including election rules, working group policies, and project roles.
+1. Steward, control access, delegate access, and establish processes regarding all Kubeflow project resources and have the final say in the disposition of those resources.
+1. Define and evolve the scope of the Kubeflow community, including acceptance of new projects into Kubeflow.
+1. Receive and handle reports about code of conduct violations and maintain confidentiality.
+1. Act as the final escalation point and arbitrator for any disputes, issues, clarifications, or escalations within the project scope.
+
+## Delegated authority
+
+KSC may choose to delegate its authority to other committees as-needed.
+
+The committee currently recognizes this delegated authority for [To be set up]:
+
+-  Technical guidance is delegated to the Technical Oversight Committee.
+-  Trademark guidance is delegated to the Conformance Committee.
+
+## Committee Meetings
+
+KSC currently meets **<frequency TBD>**, or as-needed. Meetings are open to the public and held online, unless they pertain to sensitive or previleged matters. Examples of such matters are:
+- Privacy related issues
+- Private emails to the committee
+- Code of conduct violations
+- Certain Escalations
+- Disputes between members
+- Security reports
+
+Meeting notes are available to members of the kubeflow-discuss mailing list, unless community member privacy requires otherwise. Public meetings will be recorded and the recordings made available publicly.
```

Comment:
These are currently private. So we should probably default to private and can be made public upon request?

Reply:
Edit: we decided in KSC meeting that this will be public.